tests/unit/test_column_noise.py

```diff
@@ -67,14 +67,16 @@ def dummy_dataset():
         "",
     ]
     string_series = pd.Series(string_list * int(num_simulants / len(string_list)))
-
+    zipcodes = ["12345", "98765", "02468", "13579", ""]
+    zipcode_series = pd.Series(zipcodes * int(num_simulants / len(zipcodes)))
```

Comment:
You may have noticed there's kind of an unwritten requirement that the number of simulants be divisible by the thing length of the dummy values. It may still work, it will just introduce NaNs if it isn't.

Reply:
Yup.